vowpalwabbit/example_predict.cc

```diff
@@ -7,13 +7,21 @@ license as described in the file LICENSE.
 #include "example_predict.h"
 
 safe_example_predict::safe_example_predict()
-{
-  indices = v_init<namespace_index>();
+{ indices = v_init<namespace_index>();
+  ft_offset = 0;
   // feature_space is initialized through constructors
 }
 
 safe_example_predict::~safe_example_predict()
 {
   indices.delete_v();
-  for (size_t i = 0; i < 256; i++) feature_space[i].delete_v();
+  for (size_t i=0;i<256;i++)
```

Comment:
Can we make this a constant instead of magic number?

Reply:
Fixed.

---
In reply to: [270506337](https://github.com/VowpalWabbit/vowpal_wabbit/pull/1800#discussion_r270506337) [](ancestors = 270506337)